{"name": "Неуспешный логин с неверными данными", "status": "passed", "start": 1733246972620, "stop": 1733246972795, "uuid": "dff0d494-5857-423f-811d-48537b70ca97", "historyId": "875475173a8d8fcaadc0586d31e090c2", "testCaseId": "875475173a8d8fcaadc0586d31e090c2", "fullName": "tests.test_login.TestLogin#test_login_failure", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "97619-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}